ansible/configs/sap-smart/post_software.yml

```diff
@@ -0,0 +1,198 @@
+- name: Step 00xxxxx post software
+  hosts: support
+  gather_facts: False
+  become: yes
+  tasks:
+    - debug:
+        msg: "Post-Software tasks Started"
+
+- name: Make Ansible Tower License file available
+  hosts: localhost
+  gather_facts: False
+  become: false
+  tasks:
+
+    - name: Dump Ansible Tower License to a file
+      copy:
+        content: "{{ tower_license }}"
+        dest: /tmp/license.json
+      when: tower_instance_count > 0
+
+- name: Configure Ansible Tower
+  hosts: towers
+  gather_facts: False
+  become: yes
+  tasks:
+
+    - block:
+
+      - name: Ensure dependencies for tower-cli are installed
+        dnf:
+          name: python3-pip
+          state: latest
+
+      - name: Ensure tower-cli is installed
+        pip:
+          name: ansible-tower-cli
+          state: latest
+          executable: pip3
+
+      - name: Dump tower-cli config
+        copy:
+          content: "{{ tower_cli_config }}"
+          dest: /etc/tower/tower_cli.cfg
+
+      - name: Dump Ansible Tower Workflow Schema to a file
+        copy:
+          content: "{{ workflow_schema }}"
+          dest: /tmp/schema.yml
+
+      - name: Ensure Tower License is configured
+        include_role:
+          name: infra-ansible/roles/ansible/tower/config-ansible-tower-license
+
+      - name: Ensure required Credentials are added to Tower
+        include_role:
+          name: infra-ansible/roles/ansible/tower/manage-credentials
+
+      - name: Ensure required inventories are added to Tower
+        include_role:
+          name: infra-ansible/roles/ansible/tower/manage-inventories
+
+      - name: Ensure required Projects are added to Tower
+        include_role:
+          name: infra-ansible/roles/ansible/tower/manage-projects
+
+      - name: Ensure required Job Templates are added to Tower
+        include_role:
+          name: infra-ansible/roles/ansible/tower/manage-job-templates
+
+      - name: Ensure required WorkFlow Templates are added to Tower
+        shell: /usr/local/bin/tower-cli workflow create --name="SAP HANA and S/4HANA E2E deployment"
+
+      - name: Populate new WorkFlow with specified schema
+        shell: /usr/local/bin/tower-cli workflow schema "SAP HANA and S/4HANA E2E deployment" @/tmp/schema.yml
+
+      when: tower_instance_count > 0
+
+- name: Register HANA and S4HANA hosts into Satellite and install katello agent
+  hosts: hanas:s4hanas
+  serial: 1
+  become: True
+  gather_facts: True
+  tasks:
+
+    - include_role:
+        name: redhat_sap.sap_rhsm
+
+    - name: Enable required repository for Katello Agent
+      command: subscription-manager repos --enable="satellite-tools-6.7-for-rhel-8-x86_64-e4s-rpms"
```

Comment:
Can we use `rhsm_repository` module here?

Reply:
Sure, moved!